tools/pyUANamespace/open62541_MacroHelper.py

```diff
@@ -106,8 +106,6 @@ def getCreateStandaloneReference(self, sourcenode, reference):
 
     if reference.isForward():
       code.append("UA_Server_addReference(server, " + self.getCreateNodeIDMacro(sourcenode) + ", " + self.getCreateNodeIDMacro(reference.referenceType()) + ", " + self.getCreateExpandedNodeIDMacro(reference.target()) + ", true);")
-    else:
-      code.append("UA_Server_addReference(server, " + self.getCreateNodeIDMacro(sourcenode) + ", " + self.getCreateNodeIDMacro(reference.referenceType()) + ", " + self.getCreateExpandedNodeIDMacro(reference.target()) + ", false);")
```

Comment:
As far as I can overview it, i would say yes. If the bootstrapping method is using `UA_Server_addReference`, it should also be there enough to execute it for the forward references.